backup_github.py

```diff
@@ -31,114 +40,148 @@ def __enter__(self):
-                              "the repository {}, skipping it".format(repo.name)
-                    logging.error("There was an error fetching the branches from " \
-                              "the repository {}, skipping it".format(repo.name))
+                    logger.error("There was an error fetching the branches "
+                                 "from the repository {}, "
+                                 "skipping it".format(repo.name))
                     pass
-	    logging.info("Finished copying repo {}".format(repo.name))
-	    print "Finished copying repo {}".format(repo.name)
-        #Otherwise clone the repository and fetch all branches
+            logger.info("Finished copying repo {}".format(repo.name))
+        # Otherwise clone the repository and fetch all branches
         else:
-            logging.info("The repository {} isn't cloned at {}, cloning instead of updating...".format(repo.name, repo_path))
+            logger.info("The repository {} isn't cloned at {}, cloning instead"
+                        " of updating...".format(repo.name, repo_path))
             try:
-                check_call(['git', 'clone', source, repo_path], stdout=PIPE, stderr=STDOUT)
-                logging.info("Cloning {}".format(repo.name))
-            except CalledProcessError:
-                print 'ERROR: Problem cloning repository {}, skipping it'.format(repo.name)
-                logging.error('ERROR: Error cloning repository {}, skipping it'.format(repo.name))
+                check_call(['git', 'clone', source, repo_path],
+                           stdout=PIPE, stderr=STDOUT)
+                logger.info("Cloning {}".format(repo.name))
+            except CalledProcessError as e:
+                logger.error("ERROR: Error cloning repository {}, "
+                             "skipping it".format(repo.name))
+                logger.error(str(e))
             pass
             try:
                 with cd(repo_path):
-                    check_call(track_all_branches, shell=True, stdout=PIPE, stderr=STDOUT)
-                    logging.info("Fetching branches for {}".format(repo.name))
-            except CalledProcessError:
-                print 'ERROR: Problem fetching branches for repository {}, skipping it'.format(repo.name)
-                logging.error('ERROR: Problem fetching branches for repository {}, skipping it'.format(repo.name))
+                    check_call(track_all_branches, shell=True,
+                               stdout=PIPE, stderr=STDOUT)
+                    logger.info("Fetching branches for {}".format(repo.name))
+            except CalledProcessError as e:
+                logger.error("ERROR: Problem fetching branches for "
+                             "repository {}, skipping it".format(repo.name))
+                logger.error(str(e))
                 pass
 
-def compressAndMove(source):
+
+def compressAndMove(source, final_dest):
```

Comment:
For pep8: function names should be lowercase, with words separated by underscores

Reply:
Fixed!